{{ cookiecutter.__package_name_kebab_case }}/.pre-commit-config.yaml

```diff
@@ -76,21 +81,9 @@ repos:
         args: [--check-sourced]
         language: system
         types: [shell]
-      {%- endif %}
-      - id: poetry-check
-        name: poetry check
-        entry: poetry check
-        language: system
-        files: pyproject.toml
-        pass_filenames: false
-      - id: poetry-lock-check
-        name: poetry lock check
```

Comment:
--check: Verify that poetry.lock is consistent with pyproject.toml. (Deprecated) Use poetry check --lock instead. https://python-poetry.org/docs/cli/#check